src/config/config.js

```diff
@@ -0,0 +1,46 @@
+require("dotenv").config();
+
+const {
+  DEV_DB_USERNAME,
+  DEV_DB_PASSWORD,
+  DEV_DB_NAME,
+  DEV_DB_HOSTNAME,
+  DEV_DB_PORT,
+  TEST_DB_USERNAME,
+  TEST_DB_PASSWORD,
+  TEST_DB_NAME,
+  TEST_DB_HOSTNAME,
+  TEST_DB_PORT,
+  PROD_DB_USERNAME,
+  PROD_DB_PASSWORD,
+  PROD_DB_NAME,
+  PROD_DB_HOSTNAME,
+  PROD_DB_PORT
+} = process.env;
```

Comment:
I don't think we need this part. calling `config()` should load all of the variables. Otherwise every time something new is added we have to add it here.